src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorStandalone.CSharp/Program.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace BlazorStandalone.CSharp
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
```

Comment:
Although the `Program` class should be `public` for consistency with `BlazorHosted`, the `Main` method should not (there's no need). If you could revert the change on line 10, we can merge the change on the other line :)